ansible_base/authentication/utils/claims.py

```diff
@@ -261,6 +262,10 @@ def update_user_claims(user, database_authenticator, groups):
 
     needs_save = False
     authenticator_user, _ = AuthenticatorUser.objects.get_or_create(provider=database_authenticator, user=user)
+    # update the auth_time field to align with the general format used for other authenticators
+    authenticator_user.extra_data = {**authenticator_user.extra_data, "auth_time": DateTimeField().to_representation(now())}
+    authenticator_user.save()
+
```

Comment:
@AlanCoding This might not be very efficient but couldn't figure out how to include it in the initial call on the AuthenticatorUser on line 264. I'm open for any feedback on this, tbh.

Reply:
The reason for this makes sense to me. It would be good to add `update_fields=["extra_data"]` to help a little with efficiency/isolation.

Reply:
@AlanCoding Updated the PR.